js/remove_anchor_from_url.js

```diff
@@ -5,4 +5,9 @@ function removeUrlAnchor(url) {
     return urlSplitted[0];
 }
 
-removeUrlAnchor('www.codewars.com#about')
\ No newline at end of file
+function removeAfterHash(s){
+      return s.replace(/\#.*/,'');
+}
+
+module.exports = removeUrlAnchor;
+module.exports = removeAfterHash;
```

Comment:
This overwrites the previously exported function 😄 You need to export it in another way